src/v/v8_engine/script.h

```diff
@@ -44,6 +45,7 @@ class script_exception final : public std::exception {
 class script {
     // Timeout for run script in first time for initialization
     static constexpr std::chrono::milliseconds _first_run_timeout_ms{500};
+    static constexpr uint64_t _max_old_gen_size{10_MiB};
```

Comment:
stylistically we generally don't prefix `_` on configuration options like this even though they are technically private members.